test-keys/unsafe.nts.client.toml

```diff
@@ -7,10 +7,10 @@ observation-path = "/run/ntpd-rs/observe"
 [[peer]]
 mode = "nts"
 address =  "localhost:4460"
-certificates = "test-keys/testca.pem"
+certificate_authority = "test-keys/testca.pem"
```

Comment:
I thought we didn't use underscores in field names any more?

Reply:
Suggest fixing this in separate pull request, made #1010